packages/manager/apps/public-cloud/src/assets/theme/dark/index.less

```diff
@@ -149,7 +149,10 @@
     background-color: transparent;
   }
 
-  .oui-navbar .oui-navbar-menu_toggle .oui-navbar-menu__header .oui-navbar-menu__title {
+  .oui-navbar
```

Comment:
Just one change in this file, check https://github.com/ovh/manager/pull/5350/commits/9b967982de5de4f246d98e29af01e36f3a4125ae?diff=split&w=1#diff-3879749d155d118ea8026720a806603fca4f7a809149477d867a320fd7c68c02R566.

The rest is just Prettier's work